From acdab0aecddf830d1be387f29910fc598dec65a8 Mon Sep 17 00:00:00 2001
From: Guodong Xu <guodong.xu@linaro.org>
Date: Tue, 31 Mar 2015 15:54:01 +0800
Subject: [PATCH] defconfig: hikey: discard CFG80211 and MAC80211

In TI WiFi driver R8.5, we are using backport'ed CFG80211 and MAC80211
So there is no need to build in this kernel

Signed-off-by: Guodong Xu <guodong.xu@linaro.org>
---
 arch/arm64/configs/defconfig | 14 ++------------
 1 file changed, 2 insertions(+), 12 deletions(-)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 4abfee3..77f01b9 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -168,16 +168,6 @@ CONFIG_BT_HIDP=m
 CONFIG_BT_HCIUART=m
 CONFIG_BT_HCIUART_H4=y
 CONFIG_BT_WILINK=m
-CONFIG_CFG80211=m
-CONFIG_NL80211_TESTMODE=y
-# CONFIG_CFG80211_DEFAULT_PS is not set
-CONFIG_CFG80211_DEBUGFS=y
-CONFIG_CFG80211_INTERNAL_REGDB=y
-CONFIG_CFG80211_WEXT=y
-CONFIG_MAC80211=m
-CONFIG_MAC80211_MESH=y
-CONFIG_MAC80211_LEDS=y
-CONFIG_MAC80211_DEBUGFS=y
 CONFIG_RFKILL=m
 CONFIG_RFKILL_REGULATOR=m
 CONFIG_RFKILL_GPIO=m
@@ -229,8 +219,6 @@ CONFIG_PPPOLAC=y
 CONFIG_PPPOPNS=y
 CONFIG_USB_USBNET=y
 CONFIG_WL_TI=y
-CONFIG_WL18XX=m
-CONFIG_WLCORE_SDIO=m
 CONFIG_INPUT_EVDEV=y
 CONFIG_INPUT_MISC=y
 CONFIG_INPUT_UINPUT=y
@@ -306,6 +294,7 @@ CONFIG_LEDS_CLASS=y
 CONFIG_LEDS_GPIO=y
 CONFIG_LEDS_REGULATOR=y
 CONFIG_LEDS_SYSCON=y
+CONFIG_LEDS_TRIGGERS=y
 CONFIG_LEDS_TRIGGER_HEARTBEAT=y
 CONFIG_LEDS_TRIGGER_CPU=y
 CONFIG_SWITCH=y
@@ -363,6 +352,7 @@ CONFIG_LSM_MMAP_MIN_ADDR=4096
 CONFIG_SECURITY_SELINUX=y
 CONFIG_SECURITY_SELINUX_BOOTPARAM=y
 CONFIG_SECURITY_SELINUX_DISABLE=y
+CONFIG_CRYPTO_CCM=m
 CONFIG_CRYPTO_MICHAEL_MIC=y
 CONFIG_CRYPTO_TWOFISH=y
 CONFIG_ARM64_CRYPTO=y
-- 
1.9.1

